README.md

```diff
@@ -1,17 +1,30 @@
 # Morpho Token
 
-This repository contains the Morpho protocol’s ERC20 token.
+This repository contains the Morpho protocol's ERC20 token.
 It is designed to be upgradable and support onchain delegation.
 Additionally, it ships a wrapper contract to simplify the migration of assets from the previous token contract to the new Morpho token contract.
 
-## Upgradability
+## Features
 
-The Morpho token complies with the [EIP-1967](https://eips.ethereum.org/EIPS/eip-1967) to support upgradeability.
+### Upgradeability
 
-## Delegation
+The Morpho token complies with [EIP-1967](https://eips.ethereum.org/EIPS/eip-1967) to support upgradeability.
+
+### Delegation
 
 The Morpho token supports onchain voting and voting power delegation.
 
+### Role-based Permission
```

Comment:
Consider adding a "breaking changes" section? I'd be in favor of adding a change-log including breaking changes and new features.

Reply:
I think it conflicts with this section, it would basically say the same thing. I don't think we need to put a strong emphasis on breaking changes either, because this is a new token after all (with a different address notably)

Reply:
My point is not about the content but structure. As you noted, it would not add new information but it could put forward important information. I'm not sure I grasp what you mean with conflict in this context though.

Reply:
Ok got it. Let's not add a breaking change section though because of [this comment](https://github.com/morpho-org/morpho-token-upgradeable/pull/68#discussion_r1819446244)

> I'm not sure I grasp what you mean with conflict in this context though.

Exactly what you noted: we can't have both a "Features" section and a "Breaking changes" section as they would overlap (which is what I meant by conflict)